parser/test.m

```diff
@@ -0,0 +1,40 @@
+function [output1, output2] = test(arg1, arg2)
+%TEST Computes output1 and output 2 from arg1 and arg2.
+%
+% SYNTAX
+%
+%   output1 = test(arg1, arg2)
+%   [output1, output2] = test(arg1, arg2)
+%
+% DESCRIPTION
+%
+%   Computes output1 as the sum of arg1 and arg2 and output2 as the
+%   difference between arg1 and arg2.
+%
+% INPUTS
+%
+%   arg1
+%     Scalar. This line is very long because i wanted to test if the input
+%     description would be correctly parsed if it is longer than one line.
+%
+%   arg2
+%     Scalar
+%
+% OUTPUTS
+%
+%   output1
+%     Sum of arg1 and arg2  
+%
+%   output2
+%     Difference between arg1 and arg2
+%
+% NOTES
+%
+% That function is destined to test the parsing of the function
+% documentation (done by parse_doc.m).
+
+output1 = arg1+arg2;
+% We don't want that kind of comments to appear in the parsed documentation.
```

Comment:
i don't understand this comment.  What is the purpose of this code?

Reply:
I just wanted to make sure that the comments that aren't part of the description wouldn't appear in the output structure.

Reply:
i still don't understand.
what are you trying to do here? what is the purpose of this "output1 = arg1+arg2;"?